editor/code/src/goals.ts

```diff
@@ -1,4 +1,12 @@
-import { Position, Uri, WebviewPanel, window, ViewColumn, extensions, commands } from "vscode";
+import {
```

Comment:
This appears to be unrelated

Reply:
Yes it was a formatting problem from previous PR, my bad.

It is too tiny to split IMO, so if you don't mind I'll push it here.